ccd-definition/CaseEvent/User/UserEvents-nonprod.json

```diff
@@ -1,70 +0,0 @@
-[
-  {
-    "CaseTypeID": "CIVIL",
-    "ID": "CREATE_CLAIM",
-    "Name": "Create claim",
-    "Description": "Case created, post actions triggered",
-    "DisplayOrder": 1,
-    "PostConditionState": "PENDING_CASE_ISSUED",
-    "Comment": "Event will move to PROCEEDS_IN_HERITAGE_SYSTEM if claim is raised against litigant in person",
-    "SecurityClassification": "Public",
-    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-start",
-    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-submit",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/submitted",
-    "ShowSummary": "Y",
-    "ShowEventNotes": "N",
-    "EndButtonLabel": "Submit",
-    "RetriesTimeoutAboutToStartEvent": 0,
-    "RetriesTimeoutURLAboutToSubmitEvent": 0,
-    "RetriesTimeoutURLSubmittedEvent": 0
-  },
-  {
-    "CaseTypeID": "CIVIL",
-    "ID": "WITHDRAW_CLAIM",
```

Comment:
WITHDRAW_CLAIM & DISCONTINUE_CLAIM were moved to non-prod by old team. We haven't used these any where aand don't have any context of these either. I suggest we leave these 2 under non-prod files